engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/history/SetRemovalTimeToHistoricProcessInstancesDto.ftl

```diff
@@ -16,9 +16,25 @@
   <@lib.property
       name = "hierarchical"
       type = "boolean"
-      last = true
       desc = "Sets the removal time to all historic process instances in the hierarchy.
               Value may only be `true`, as `false` is the default behavior."/>
 
+  <@lib.property
+      name = "updateInChunks"
+      type = "boolean"
+      desc = "Processes removal time updates in chunks, taking into account defined limits.
+              This can lead to multiple executions of the resulting jobs and prevents the database
+              transaction from timing out by limiting the number of rows to update.
+              Value may only be `true`, as `false` is the default behavior."/>
```

Comment:
```suggestion
      desc = "Processes removal time updates in chunks, taking into account the defined size in
              the process engine configuration. The size of the chunks can also be defined per call
              with the `updateChunkSize` parameter.
              Enabling this can lead to multiple executions of the resulting jobs, preventing the 
              database transaction from timing out by limiting the number of rows to update.
              Value may only be `true`, as `false` is the default behavior."/>
```